README.md

```diff
@@ -17,8 +17,8 @@ Load `@octokit/plugin-request-log` and [`@octokit/core`](https://github.com/octo
 
```

Comment:
you missed one on line 16 :)

Reply:
Davide, have you seen my feedback ☝🏼 We will merge the pull requests for #101 soon, you'll need to fix yours in order for it to count as contribution to the project, and as hacktoberfest contribution (in case you participate)